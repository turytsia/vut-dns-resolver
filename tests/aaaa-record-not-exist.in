-r -t -6 -s kazi.fit.vutbr.cz www.github.com