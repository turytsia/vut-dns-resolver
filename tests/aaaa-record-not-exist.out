Authoritative: No, Recursive: Yes, Truncated: No
Question section (1)
 www.github.com., AAAA, IN
Answer section (1)
 www.github.com., CNAME, IN, 0, github.com.
Authority section (1)
 www.github.com., CNAME, IN, 0, github.com.
Additional section (0)
